Editor/XRUtils/XRLoaderHelper.cs

```diff
@@ -19,6 +18,7 @@ namespace Innoactive.CreatorEditor.XRUtils
     /// </summary>
     internal class XRLoaderHelper
     {
+        internal const string AttemptedToInitialized = "IsXRLoaderInitialized";
```

Comment:
```suggestion
        internal const string AttemptedToInitialize = "IsXRLoaderInitialized";
```
I would say a better name is "IsXRLoaderInitialized", though